tests/robustness/traffic/etcd.go

```diff
@@ -64,6 +64,16 @@ var (
 			{Choice: Put, Weight: 40},
 		},
 	}
+	EtcdDelete Traffic = etcdTraffic{
```

Comment:
Tested the traffic, didn't find having `EtcdDelete` increases the reproducibility over `EtcdPutDeleteLease`. Don't think there is a need to create third type of traffic.